app/controllers/application_controller.rb

```diff
@@ -519,6 +520,20 @@ def user_is_banned?
     BannedUserResolver.new(current_user).banned_for_sp?(issuer: current_sp&.issuer)
   end
 
+  def user_duplicate_profiles_detected?
+    return false unless sp_eligible_for_one_account?
+    profile = current_user&.active_profile
+    DuplicateProfileConfirmation.where(
+      profile_id: profile.id,
+      confirmed_all: nil,
+    ).present?
+  end
+
+  def sp_eligible_for_one_account?
+    sp_from_sp_session.present? &&
```

Comment:
Hmm, I think we could do away with this check given the safe navigation in the next line?